### PR TITLE
hasher pallet tests

### PR DESCRIPTION
**Pull request by shekohex**

### CODE REVIEW

File: pallets/hasher/src/tests.rs [hunk 1]
```diff
@@ -1,14 +1,44 @@
...
 			Error::<Test, Instance1>::ParametersNotInitialized
 		);
 	});
 }
+
+#[test]
+fn should_initialize_parameters() {
+	new_test_ext().execute_with(|| {
+		let params = PoseidonParameters::<ark_bn254::Fr>::new(
+			constants::bn256_3x5::ROUND_KEYS.into(),
```
Comment (drewstone) [inline pallets/hasher/src/tests.rs:21]: Should use bn254 to be consistent, no?

**Review by drewstone — CHANGES_REQUESTED** [review]
Looks good, we should merge after fixing consistency issues.

File: pallets/hasher/src/tests.rs (hunk 1 above)
Comment (shekohex) [reply pallets/hasher/src/tests.rs]: Done

**Review by drewstone — APPROVED** [review]
Looks great!

**Pull request merged**